ui/src/views/mint/github-step/github-repo-configuration.tsx

```diff
@@ -0,0 +1,130 @@
+import {
+  Button,
+  Card,
+  Dropdown,
+  DropdownItem,
+  Form,
+  Grid,
+  Icon,
+  IconButton,
+  Stepper,
+} from '@/components';
+import React, { useState } from 'react';
+import { Mint } from '../mint.context';
+import { RepoRow } from './github-repository-selection';
+
+//TODO remove once it's integrated with GH login
+const branches: DropdownItem[] = [

```

Comment:
For this list we are probably going to need to fetch the branches, so we need to be aware of adding loading states for this. One scenario to verify is if github returns all branches at once or we will need to deal with some kind of pagination or search in the github api.

Reply:
Yeap, I thought about the loading state. I can add it now, or later when we add the github integration